doc/release_notes/release_notes_1.0.2.rst

```diff
@@ -0,0 +1,55 @@
+.. _release_notes_1.0.2:
+
+ACRN v1.0.2 (November 2019)
+#######################
+
+We are pleased to announce the release of ACRN version 1.0.2. This is a
+maintenance release based off the v1.0 branch that primarily fixes some
+potential security and stability issues discovered after the v1.0
+release.
+
+ACRN is a flexible, lightweight reference hypervisor that's built with
+real-time and safety-criticality in mind and is optimized to streamline
+embedded development through an open source platform.  Check out
+:ref:`introduction` for more information. All project ACRN source code
+is maintained in the https://github.com/projectacrn/acrn-hypervisor
+repository and includes folders for the ACRN hypervisor, the ACRN device
+model, tools, and documentation. You can either download this source
+code as a zip or tar.gz file (see the `ACRN v1.0.2 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.0.1>`_)
```

Comment:
Should be "https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.0.2"

Reply:
done